gc/base/CollectorLanguageInterface.hpp

```diff
@@ -558,6 +558,15 @@ class MM_CollectorLanguageInterface : public MM_BaseVirtual {
 #endif /* OMR_GC_MODRON_CONCURRENT_MARK */
 
 	/**
+	 * This method is called to check if vm is in Startup stage, return true if vm is not in Startup stage(Default).
+	 *
+	 * @param[in] env The environment for the calling thread.
+	 * @return true if vm is not in Startup stage.
+	 */
```

Comment:
Can you change the name and comments for this function to be isVMInStartupPhase with a return of true if the VM is in startup phase.  Of course you will have to also change the consumers.  Have the Not in the name for a boolean function makes it much harder to read and understand what is going on.  Thanks